js/modules/k6/execution/execution_test.go

```diff
@@ -186,7 +186,7 @@ func TestVUTags(t *testing.T) {
 	})
 }
 
-func TestAbortTest(t *testing.T) { //nolint: tparallel
+func TestAbortTest(t *testing.T) { //nolint:tparallel
```

Comment:
nit: I kind of prefer to not have unrelated changes in unrelated files like that ;)

Reply:
It sounds like a personal preference :thinking: Let me try to provide my two cents on why I'm doing this and continue doing it.

It's a common/best practice to leave the place cleaner than before. For example, this is a comment for a machine (linter) comment, and it doesn't require space in between. The same things I usually do when spotting any grammar issues. Such changes itself is super-minor and don't deserve a separated PR.

Does that make sense? Will it help if I'll do this in a separate commit?